lib/image.coffee

```diff
@@ -12,6 +12,10 @@ emit = ($item, item) ->
 
 bind = ($item, item) ->
   $item.dblclick -> editor.textEditor $item, item
-  $item.find('img').dblclick -> dialog.open item.text, this
+  url = item.url
+  if item.ipfs?
```

Comment:
The item url will be for the postcard image. Folks who want nothing to do with ipfs will get the same old postcards they use to get. Those who like ipfs will run local nodes and get fabulous service unavailable from public gateways.